### PR TITLE
SOV-2938 add wrbtc & iwrbtc address in feeSharing storage

### PR DESCRIPTION
**Pull request by cwsnt**

### CODE REVIEW

File: contracts/governance/FeeSharingCollector/FeeSharingCollector.sol [hunk 1]
```diff
@@ -119,6 +131,40 @@ contract FeeSharingCollector is
...
+    /**
+     * @notice Set the wrbtc token address of fee sharing collector.
+     *
+     * only owner can perform this action.
+     *
+     * @param newWrbtcTokenAddress The new address of the wrbtc token.
+     * */
+    function setWrbtcToken(address newWrbtcTokenAddress) public onlyOwner {
+        emit SetWrbtcToken(msg.sender, wrbtcTokenAddress, newWrbtcTokenAddress);
+        wrbtcTokenAddress = newWrbtcTokenAddress;
```
Comment (tjcloa) [inline contracts/governance/FeeSharingCollector/FeeSharingCollector.sol:153]: should be a contract address, cannot be 0

File: contracts/governance/FeeSharingCollector/FeeSharingCollector.sol [hunk 2]
```diff
@@ -119,6 +131,40 @@ contract FeeSharingCollector is
...
+    /**
+     * @notice Set the loan wrbtc token address of fee sharing collector.
+     *
+     * only owner can perform this action.
+     *
+     * @param newLoanTokenWrbtcAddress The new address of the loan wrbtc token.
+     * */
+    function setLoanTokenWrbtc(address newLoanTokenWrbtcAddress) public onlyOwner {
+        emit SetLoanTokenWrbtc(msg.sender, loanTokenWrbtcAddress, newLoanTokenWrbtcAddress);
+        loanTokenWrbtcAddress = newLoanTokenWrbtcAddress;
```
Comment (tjcloa) [inline contracts/governance/FeeSharingCollector/FeeSharingCollector.sol:165]: should be a contract address, cannot be 0

**Review by tjcloa — CHANGES_REQUESTED** [review]
a few improvements requested

**Review by tjcloa — CHANGES_REQUESTED** [review]
pls add scripts to create multisig txs to initialize the addresses

File: contracts/governance/FeeSharingCollector/FeeSharingCollector.sol (hunk 2 above)
Comment (cwsnt) [reply contracts/governance/FeeSharingCollector/FeeSharingCollector.sol:174]: Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/519/commits/d3e520f0166c70d672d645ac02824662abc1c874)

File: contracts/governance/FeeSharingCollector/FeeSharingCollector.sol (hunk 1 above)
Comment (cwsnt) [reply contracts/governance/FeeSharingCollector/FeeSharingCollector.sol:158]: Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/519/commits/d3e520f0166c70d672d645ac02824662abc1c874)

File: contracts/governance/FeeSharingCollector/FeeSharingCollector.sol [hunk 3]
```diff
@@ -119,6 +131,40 @@ contract FeeSharingCollector is
     /// @dev fallback function to support rbtc transfer when unwrap the wrbtc.
     function() external payable {}
 
+    /**
+     * @dev initialize function for fee sharing collector proxy
+     * @param wrbtcToken wrbtc token address
+     * @param loanWrbtcToken address of loan token wrbtc (IWrbtc)
+     */
+    function initialize(address wrbtcToken, address loanWrbtcToken) external onlyOwner {
```
[the remark this answers is not in the document]
Comment (cwsnt) [reply contracts/governance/FeeSharingCollector/FeeSharingCollector.sol]: Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/519/commits/d3e520f0166c70d672d645ac02824662abc1c874)

File: contracts/governance/FeeSharingCollector/FeeSharingCollectorStorage.sol [hunk 4]
```diff
@@ -76,6 +76,16 @@ contract FeeSharingCollectorStorage is Ownable {
 
     mapping(bytes4 => bool) public isFunctionExecuted;
 
+    /**
+     * @dev Wrbtc token adress
+     */
+    address public wrbtcTokenAddress;
+
+    /**
+     * @dev IWrbtc token adress
```
[the remark this answers is not in the document]
Comment (cwsnt) [reply contracts/governance/FeeSharingCollector/FeeSharingCollectorStorage.sol]: Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/519/commits/d3e520f0166c70d672d645ac02824662abc1c874)

**Comment by cwsnt** [comment]
> pls add scripts to create multisig txs to initialize the addresses

Added the initialization script

**Review by tjcloa — APPROVED** [review]

**Review by ororopickpocket — APPROVED** [review]

**Pull request merged**